docs/platform/integration/commercetools/ctp-client.md

```diff
@@ -0,0 +1,104 @@
+---
+id: ctp-client
+title: commercetools ctpClient
+sidebar_label: ctpClient
+enterprise_only: true
+---
+
+// TODO
+- [ ] ctpIntroduction
```

Comment:
Is this TODO for us internally or is it meant to be left in?

If it's for the customers i think we should choose a different title and leave a note by the items we don't support e.g. `- [] ctpIntroduction (In development)`